docusaurus/sidebars.js

```diff
@@ -310,6 +310,7 @@ const contributeToAirbyte = {
       items: [
         "contributing-to-airbyte/resources/pull-requests-handbook",
         "contributing-to-airbyte/resources/code-style",
+        "contributing-to-airbyte/resources/code-formatting",
         "contributing-to-airbyte/resources/developing-locally",
         "contributing-to-airbyte/resources/developing-on-docker",
         "contributing-to-airbyte/resources/gradle",
```

Comment:
Ooh can we remove the code formatting references from `/gradle` and `python-gradle-setup` too?

Reply:
Nice catch. I think the content of this doc is outdated and would deserve a full rewrite (probably on `developing-locally` and `developing-on-docker` too.
I marked these two docs as deprecated and removed them from the sidebar.